Frontend/ui-driver/src/Flow.purs

```diff
@@ -1799,13 +1799,15 @@ homeScreenFlow = do
       liftFlowBT $ logEventWithParams logField_ label "Timestamp" currentTime
       changeDriverStatus status
       gs <- getState
+      when (status == Online || status == Silent) do
+        getDriverInfoApiResp <- lift $ lift $ Remote.getDriverInfoApi (GetDriverInfoReq{})
+        case getDriverInfoApiResp of
```

Comment:
Can we use `either` function and remove case match here -

https://pursuit.purescript.org/packages/purescript-either/6.1.0/docs/Data.Either#v:either